docs/cluster-configuration.md

```diff
@@ -22,6 +22,8 @@ cluster = Cluster(ClusterConfiguration(
     image="quay.io/project-codeflare/ray:latest-py39-cu118", # Mandatory Field
     instascale=False, # Default False
```

Comment:
I think this line relating to Instascale should be removed while we are updating this file - please

Reply:
Removed